ggshield/verticals/secret/rich_secret_scanner_ui.py

```diff
@@ -27,6 +32,9 @@ def __exit__(self, *args: Any) -> None:
         self.progress.__exit__(*args)
 
     def on_scanned(self, scannables: Sequence[Scannable]) -> None:
+        if self.verbose:
+            for scannable in scannables:
+                self.progress.console.print(f"Scanned [blue]{scannable.path}")
```

Comment:
For consistency, can you change the message string in `on_skipped()` to use `[blue]` for the path as well (or not use `[blue]` and let Rich do its magic, whichever you prefer). Otherwise it looks odd:

![image](https://github.com/GitGuardian/ggshield/assets/91945295/b6e023fa-b00b-4353-ab3b-ab8138df9bbf)

Reply:
I've removed the `[blue]`.